examples/echo.rs

```diff
@@ -91,6 +88,9 @@ fn main() {
     let (lossless_tx, lossless_rx) = mpsc::unbounded();
     let (lossy_tx, lossy_rx) = mpsc::unbounded();
 
+    let (friend_request_sink_tx, friend_request_sink_rx) = mpsc::unbounded();
```

Comment:
The word `sink` is redundant here. It's a (sink, stream) tuple or in other words (tx, rx).